src/sdc11073/xml_types/xml_structure.py

```diff
@@ -477,53 +548,69 @@ def update_xml_value(self, instance, node):
 
```

Comment:
line 546: `Unresolved attribute reference 'elem_to_xml' for class 'DataConverterProtocol' `

Reply:
I think this cannot be expressed with typing. The dataconverter is a ListConverter in __init__,  which has this method.